test/Microsoft.Azure.WebJobs.Host.UnitTests/Loggers/ApplicationInsightsLoggerTests.cs

```diff
@@ -87,8 +88,8 @@ public async Task LogFunctionResult_Succeeded_SendsCorrectTelemetry()
             string expectedOperationId, expectedRequestId;
             using (logger.BeginFunctionScope(CreateFunctionInstance(_invocationId), _hostInstanceId))
             {
-                expectedRequestId = Activity.Current.Id;
-                expectedOperationId = Activity.Current.RootId;
+                expectedRequestId = $"|{Activity.Current.GetTraceId()}.{Activity.Current.GetSpanId()}.";
+                expectedOperationId = Activity.Current.GetTraceId();
```

Comment:
Curious: Did these change with the move to the new ApplicationInsights package?

Reply:
These are extension methods that implement W3C distributed tracing. They've been added in ApplicationInsights v2.8.
W3C support is being implemented in .NET Core and we'll consume them in ApplicaitonInsights after 3.0 releases instead of these extensions methods.